examples/with-ava/src/index.test.js

```diff
@@ -0,0 +1,7 @@
+import test from 'ava'
+import { sum } from './index'
+
+test('sum', t => {
+    const result = sum(2, 5)
+    t.is(result, 7)
+})
```

Comment:
This PR can't be merged unless you rebase it because of the accidental inclusion of build artifacts. It is probably easier to close and redo than go through rebasing (if you're not super familiar with it).